app/src/main/java/com/devshish/internship/presentation/ui/player/PlayerViewModel.kt

```diff
@@ -28,10 +34,22 @@ class PlayerViewModel(
         get() = _playbackPosition
     private val _playbackPosition = MutableLiveData<String>()
 
+    val searchSong: LiveData<SearchSong>
+        get() = _searchSong
+    private val _searchSong = MutableLiveData<SearchSong>()
+
+    val navigateToLyricsEvent: LiveData<Event<SearchSong>>
+        get() = _navigateToLyricsEvent
+    private val _navigateToLyricsEvent = MutableLiveData<Event<SearchSong>>()
+
     init {
         mediaBrowser.currentSongCallback = object : MediaBrowserClient.CurrentSongCallback {
             override fun updateSong(song: Song) {
                 _songToPlay.value = song
+
+                viewModelScope.launch {
+                    _searchSong.value = repository.searchSongs(song.title).first()
```

Comment:
1. Leave here //TODO comment, because we right now hardcode some logic
2. Also, how about make  searchSong: LiveData<SearchSong> private and make just isLirycsButtonVisible